cmd/skill.go

```diff
@@ -73,10 +96,20 @@ Missile Weapons      {{.Missile}} ({{.MissileTotal}}/{{.MissileNext}})
 		strconv.Itoa(config.WeaponExpNext(s.actor.Skills[4].Value, s.actor.Class)),
 		strconv.Itoa(s.actor.Skills[5].Value),
 		strconv.Itoa(config.WeaponExpNext(s.actor.Skills[5].Value, s.actor.Class)),
+		strconv.Itoa(s.actor.Skills[6].Value),
+		strconv.Itoa(config.WeaponExpNext(s.actor.Skills[6].Value, s.actor.Class)),
+		strconv.Itoa(s.actor.Skills[7].Value),
+		strconv.Itoa(config.WeaponExpNext(s.actor.Skills[7].Value, s.actor.Class)),
+		strconv.Itoa(s.actor.Skills[8].Value),
+		strconv.Itoa(config.WeaponExpNext(s.actor.Skills[8].Value, s.actor.Class)),
+		strconv.Itoa(s.actor.Skills[9].Value),
+		strconv.Itoa(config.WeaponExpNext(s.actor.Skills[9].Value, s.actor.Class)),
 	}
 
 	if s.actor.Class == 8 {
 		skill_header += monk_skills
+	} else if s.actor.Class == 4 {
```

Comment:
move this as a solo if statement after the regular statement.  Mages should display both weapon and elemental affinity not only elemental.